docs/ocis/deployment/ocis_oc10_backend.md

```diff
@@ -9,8 +9,10 @@ geekdocFilePath: ocis_frontend_oc10_backend.md
 
 {{< toc >}}
 
-This deployment scenario shows how to use ocis as frontend for a existing owncloud 10 installation.
-ocis will allow owncloud 10 users to log in and work with their files.
+This deployment scenario shows how to use ocis as frontend for an existing ownCloud 10 installation.
+ocis will allow ownCloud 10 users to log in and work with their files. While the deployment scenario includes
+an ownCloud 10 instance, it only exists to show the necessary configuration for your already existing ownCloud 10
+installation.
```

Comment:
so is this a development scenario? or a production scenario? if it can be used for both we need a section that describes what needs to be changed for a production deployment, like removing the http endpoints above.

Reply:
Ok, so I misread some of the above. It nowhere mentions `development`. In any case maybe rephrase as
> This deployment scenario shows how to use ocis as frontend for an existing _productive_ ownCloud 10 installation.

Reply:
And s/productive ownCloud 10 installation/production ownCloud 10 installation/

Reply:
:heavy_check_mark: